src/github.com/getlantern/flashlight/status_pages/generic_error.html

```diff
@@ -0,0 +1,115 @@
+
+      <h1>Address not found</h1>
+
+      <strong class="short-error">Lantern can’t find the server at {{.ServerName}}.</strong>
```

Comment:
I would maybe just make this a little more generic. Since we don't know what the error was at this point, can we just say "Lantern could not access the server at {{.ServerName}}."?